src/main/payment/main/G1HRPS/Payment.java

```diff
@@ -1,164 +1,277 @@
 package main.G1HRPS;
 
 import java.util.UUID;
+import java.util.List;
 
 public class Payment {
 	/**
 	 * Unique generated ID
 	 */
 	private final UUID payment_id_;
-	private int guest_id_;
+	private String guest_id_;
+	private int room_num_; 
 	private float discounts_;
 	private float tax_;
+	private float room_charges_;
+	private float rs_charges_;
```

Comment:
Please use clearly understandable naming for variables, i.e. room_service_charges_

Reply:
I'll resolve the getters description and unclear var/method names